content/docs/command-reference/remote/modify.md

```diff
@@ -663,7 +663,7 @@ a specific user. Please refer to
 more information.
 
 - `credentialpath` - path to the file that contains the
-  [service account key](/doc/user-guide/setup-google-drive-remote#using-service-accounts).
+  [service account key](https://cloud.google.com/docs/authentication/getting-started#creating_a_service_account).
```

Comment:
> > GCP Cloud shouldn't really link to Google Drive setup.
> 
> Where do we link to GDrive setup? I don't see that changed in this PR.

Here.

Reply:
I see. Makes sense to make the link more direct here.